docs/tutorial.md

```diff
@@ -87,13 +89,15 @@ RIG responds with `202 Accepted`, followed by the CloudEvent as sent to subscrib
 
 > If there are no subscribers for a received event, the response will still be `202 Accepted` and the event will be silently dropped.
 
-## 5. The event has been delivered to our subscriber
+## 5. The event has been delivered to our subscriber [Frontend]
+
+Going back to the first terminal window you should now see your greeting event
 
-Going back to the first terminal window you should now see your greeting event :tada:
+# Connect your app to RIG
 
-## 6. Connect your app to RIG
+In a real-world frontend app the above example to connect your app to RIG would look something like this below.
```

Comment:
thought you wanted to get rid of the code below as well.. no longer the case?

Reply:
I wanted to keep, but for me it was just a bit misleading as it was numbered :)